crates/fs/common/metadata.rs

```diff
@@ -31,7 +33,7 @@ pub struct Metadata(pub BTreeMap<String, Ipld>);
 impl Metadata {
     /// Creates a new metadata representing a UnixFS node.
     pub fn new(time: DateTime<Utc>) -> Self {
-        let time = time.timestamp();
+        let time = time.timestamp_nanos();
```

Comment:
Why `timestamp_nanos` now?
I'd rather have them be POSIX-compatible second-based timestamps, which is probably what most people familiar with POSIX interfaces would expect (especially since we don't have anything indicating the unit here).
It's especially important for JS, where 64-bit integers aren't a thing.

Reply:
Wasn't aware of this constraint. Let me fix that.
Was having troubles converting `time.timestamp()` back to `Utc`. Chronos seems to represent time to its nanoseconds, and that was  breaking the equality checks.